src/bundle/Resources/views/admin/policy/list.html.twig

```diff
@@ -25,9 +25,9 @@
                 <td>{{ policy.function }}</td>
                 <td>
                     {% for limitation in policy.limitations %}
-                        {{ limitation.identifier }}
-
-                        ({{ limitation.limitationValues | join(',') }})
+                        <strong>{{ limitation.identifier }}</strong>:
```

Comment:
`<strong>` tag has own meaning, which is "this is more important than other text". I think it's not that case.
I suggest using just `<span>` with css class to bold text.

Reply:
Another option would be using `<b></b>` tag :)